smartsim/settings/pbsSettings.py

```diff
@@ -185,40 +213,71 @@ def format_batch_args(self) -> t.List[str]:
             opts += [" ".join((prefix + opt, str(value)))]
         return opts
 
+    def _sanity_check_resources(
+            self,
+            resources: t.Optional[t.Dict[str, t.Union[str,int]]] = None
+    ) -> None:
```

Comment:
Very pedantic black style ask:
```suggestion
    def _sanity_check_resources(
        self,
        resources: t.Optional[t.Dict[str, t.Union[str,int]]] = None,
    ) -> None:
```

Reply:
There's a couple of other small format-y things. Can you run black over this file before we merge?

Reply:
beat me to `black` also!